test/basic-test.js

```diff
@@ -1,11 +1,11 @@
-import * as assert from "assert";
-import sift, { indexOf as siftIndexOf } from "..";
+import * as assert from 'assert';
```

Comment:
Just prettier changes here?

Reply:
Yup. I should've separated out the prettier changes but by the time I realized that I'd already committed :|